### ISSUE TITLE
Increase squashing window text space

### ISSUE THREAD

**Issue by Adrianilloo** [desktop/desktop#13018]
I've been using the GUI based squashing feature for enough time already and noticed a user experience limitation of moderate impact, sometimes.

The text area that initially contains all involved commits' messages is too small, preventing user to view the whole text at once when enough length is involved, and scrolling doesn't help that for obvious reasons.

The ideal solution, would be, overall, the following two measures:

- Set initial text area size auto-computed between a minimum (for instance, the current size) and a max. (e.g. 2x the current size), depending on the vertical exact space that initial text makes.
- Add a grabber to the bottom-right corner, as HTML 5 `textarea` has (this is the most important change towards fixing the issue). Less importantly, it would probably be fine to keep controllable size to the minimum mentioned above. Then, I guess the horizontal space that the grabber creates should be a new right margin for text (as the below picture illustrates).

![Squashing](https://user-images.githubusercontent.com/15228896/134766980-71779ae9-7429-4d24-b625-f8aca6972b67.PNG)

Thank you!

**Comment by JimmyCushnie** [comment]
Yes please, this would make squashing a lot better. When squashing commits, you typically want to have fairly long descriptions, since you're covering the changes from several commits in a single commit message. There's no reason to have such a small window imo.
